cmd/podman/machine/create.go

```diff
@@ -0,0 +1,107 @@
+package machine
+
+import (
+	"github.com/containers/common/pkg/completion"
+	"github.com/containers/podman/v3/cmd/podman/registry"
+	"github.com/containers/podman/v3/pkg/domain/entities"
+	"github.com/containers/podman/v3/pkg/machine"
+	"github.com/containers/podman/v3/pkg/machine/qemu"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+)
+
+var (
+	createCmd = &cobra.Command{
+		Use:               "create [options] NAME",
```

Comment:
You are not creating a VM, you are initializing a VM, I don't think we should use create for this.
I was thinking it should be init, but i am willing to listen to alternatives.

Reply:
I'm in the process of renaming this